### ISSUE TITLE
Serialization of single object with context / options

### ISSUE THREAD

**Issue by bbay** [panko-serializer/panko_serializer#83]
We already discussed serialization of a single object and the docs were updated with the example [here](https://panko.dev/docs/response-bag).

Now, I face the situation where I need to pass some options and I noticed it is not possible due to the implementation in ResponseCreator ([response.rb](https://github.com/panko-serializer/panko_serializer/blob/master/lib/panko/response.rb))

wdyt about adding this change?
```ruby
   # in lib/panko/response.rb
    def self.serializer(data, serializer, options = {})
      json serializer.new(options).serialize_to_json(data)
    end
```
So I can then do something like this:
```ruby
render(
  json: Panko::Response.create do |r|
    {
      status: 'active',
      user: r.serializer(user, UserSerializer, context: {index: true})
    }
  end
)
```

I could send a Pull Request if you think this is a good idea.

**Comment by yosiat** [comment]
Hi @bbay ,

Sounds good, I am always open to PR :)

**Comment by bbay** [comment]
Closed at #84 
Thanks !! Waiting for that fancy preloader now! 😍
